pkgs/by-name/so/socalabs-papu/package.nix

```diff
@@ -0,0 +1,171 @@
+{
+  stdenv,
+  fetchFromGitHub,
+  lib,
+  cmake,
+  pkg-config,
+  alsa-lib,
+  copyDesktopItems,
+  makeDesktopItem,
+  xorg,
+  freetype,
+  expat,
+  libGL,
+  libjack2,
+  curl,
+  webkitgtk_4_0,
+  libsysprof-capture,
+  pcre2,
+  util-linux,
+  libselinux,
+  libsepol,
+  libthai,
+  libxkbcommon,
+  libdatrie,
+  libepoxy,
+  libsoup_2_4,
+  lerc,
+  sqlite,
+  ninja,
+  # Disable VST building by default, since NixOS doesn't have a VST license
+  enableVST2 ? false,
+}:
+stdenv.mkDerivation {
+  pname = "socalabs-papu";
+  version = "1.1.0";
+
+  src =
+    (fetchFromGitHub {
+      owner = "FigBug";
+      repo = "PAPU";
+      rev = "e7c42c7d9056f21ec5bbdcb101908969effe9db0";
```

Comment:
any way to use tag + a version here instead?

Reply:
Nope - no tags in the original repo.